packages/ui/src/popovers/popover.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { Fragment, useState } from "react";
+import { usePopper } from "react-popper";
+import { Popover as HeadlessReactPopover, Transition } from "@headlessui/react";
+// helpers
+import { cn } from "../../helpers";
+// types
+import { TPopover } from "./types";
+import { EllipsisVertical } from "lucide-react";
+
+export const Popover = (props: TPopover) => {
+  const {
+    popperPosition = "bottom-end",
+    popperPadding = 0,
+    buttonClassName = "",
```

Comment:
Since they have an option to send a button, lets have a sane default for button and not take it as a prop.